src/avfoundation.cs

```diff
@@ -9509,7 +9509,7 @@ enum AVCaptureDeviceType {
 		[Field ("AVCaptureDeviceTypeBuiltInTelephotoCamera")]
 		BuiltInTelephotoCamera,
 
-		[iOS (10, 0, message: "Use 'BuiltInDualCamera' instead.")]
+		[iOS (10, 0)]
```

Comment:
I checked, the original availability had this. PMCS ignores it but without it we error.